src/app/qgsattributesformproperties.cpp

```diff
@@ -941,6 +981,9 @@ QTreeWidgetItem *DnDTree::addItem( QTreeWidgetItem *parent, QgsAttributesFormPro
       case QgsAttributesFormProperties::DnDTreeItemData::HtmlWidget:
         //no icon for HtmlWidget
         break;
+
+      case QgsAttributesFormProperties::DnDTreeItemData::WidgetType:
```

Comment:
This will never happen since this is all in the other if clause checking for container (old thing I mentioned in the last pr). Maybe you are keeping it for your upcoming implementations.

Anyway, I would check here for WidgetType and add this `newItem->setBackground( 0, QBrush( Qt::lightGray ) );`

Reply:
I will take care of the icons later.
But sadly the svg are missing.

Reply:
sorry, the fix for the light gray is coming in next round